src/main/java/net/imglib2/type/numeric/integer/IntegerTypes.java

```diff
@@ -0,0 +1,42 @@
+package net.imglib2.type.numeric.integer;
+
+import net.imglib2.type.logic.BitType;
+import net.imglib2.type.numeric.IntegerType;
+
+public class IntegerTypes
+{
+	private IntegerTypes()
+	{
+		// NB: prevent instantiation of static utility class
+	}
+
+	public static IntegerType< ? > smallestType( boolean signed, long max )
```

Comment:
Add JavaDoc to clarify what is meant by `smallest`.

Reply:
Done. Happy to hear suggestions how to improve/clarify though 🙂 .

Reply:
I like your JavaDoc, this is resolved for me